scripts/release.mjs

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env zx
+import { argv, cwd } from 'process'
+
+await $`set -e`
+
+const DIR = __dirname
+
+await $`zx ${DIR}/grabthar-validate-git`
+await $`zx ${DIR}/grabthar-validate-npm`
```

Comment:
I'm looking into how to run `zx` scripts without getting "Permission denied" when not explicitly using the `zx` executable. I'm also looking into why I currently need to include the `.mjs` extension to avoid "No such file or directory" when running a `zx` script:

![Screen Shot 2022-02-02 at 11 26 19 AM](https://user-images.githubusercontent.com/20399044/152216198-72e727de-f848-4273-8c91-2cc8f99461d1.png)

Reply:
I went ahead and made `release.mjs` executable by anyone, and don't need to worry about the extensions once the scripts are in `some-paypal-repo/node_modules/bin`.